app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -399,7 +434,7 @@ function AcuantCapture(
         capture={capture}
         value={value}
         errorMessage={ownErrorMessage ?? errorMessage}
-        onClick={startCaptureOrTriggerUpload}
+        onClick={withLoggedClick('placeholder')(startCaptureOrTriggerUpload)}
```

Comment:
Looking at this...it seems like we could just make the function we're wrapping an argument? And then immediately return the wrapped function?

```suggestion
        onClick={withLoggedClick('placeholder', startCaptureOrTriggerUpload)}
```

and I think we could simplify the implementation/templating too?

Reply:
> Looking at this...it seems like we could just make the function we're wrapping an argument? And then immediately return the wrapped function?

Technically, yes. Conventionally, I'd tend to treat these `with`-prefixed functions as curried, though admittedly there's not a ton of usefulness to doing so here, since we're not reusing a click source.